src/synology_dsm/api/dsm/information.py

```diff
@@ -51,9 +51,9 @@ def serial(self) -> str:
         return self._data["serial"]
 
     @property
-    def temperature(self) -> int:
+    def temperature(self) -> int | None:
         """Temperature of the NAS."""
-        return self._data["temperature"]
+        return self._data.get("temperature", None)

```

Comment:
```suggestion
        return self._data.get("temperature")
```

`.get()` returns `None` per default when the requested key is not found